docs/docs/APIGuide/Layers/Convolution-Layers.md

```diff
@@ -802,6 +802,124 @@ output m is : [[[-0.04712385  0.21949144  0.0843184   0.14336972]
+```
+**Python:**
+```python
+m = SpatialFullConvolution(n_input_channel, n_output_channel, depth_multiplier, kernel_w, kernel_h, stride_w = 1, stride_h = 1, pad_w = 0, pad_h = 0, with_bias = True, data_format="NCHW", w_regularizer=None, b_regularizer=None, p_regularizer=None)
+```
+
+Separable convolutions consist in first performing a depthwise spatial convolution (which acts
+on each input channel separately) followed by a pointwise convolution which mixes together the
+resulting output channels. The  depthMultiplier argument controls how many output channels are
+enerated per input channel in the depthwise step.
+
+ * `nInputChannel` The number of expected input planes in the image given into forward()
+ * `nOutputChannel` The number of output planes the convolution layer will produce.
+ * `depthMultiplier` how many internal channels are generated per input channel
+ * `kW` The kernel width of the convolution.
+ * `kH` The kernel height of the convolution.
+ * `sW` The step of the convolution in the width dimension.
+ * `sH` The step of the convolution in the height dimension.
+ * `pW` The additional zeros added per width to the input planes. Default is 0.
+ * `pH` The additional zeros added per height to the input planes. Default is 0.
+ * `hasBias` do we use a bias on the output
+ * `dataFormat` image data format, which can be NHWC or NCHW
+ * `wRegularizer` kernel parameter regularizer
+ * `bRegularizer` bias regularizer
+ * `pRegularizer` point wise kernel parameter regularizer
+ * `initDepthWeight` kernel parameter init tensor
+ * `initPointWeight` point wise kernel parameter init tensor
+ * `initBias` bias init tensor
+ 
+**Scala example:**
+
+```scala
+
+scala>
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric.NumericFloat
+import com.intel.analytics.bigdl.nn._
+import com.intel.analytics.bigdl.tensor._
+import com.intel.analytics.bigdl.nn.abstractnn.DataFormat
+
+val m = SpatialSeperableConvolution[Float](1, 2, 1, 2, 2, dataFormat = DataFormat.NCHW)
+val input = Tensor(1, 1, 3, 3).randn()
+val output = m.forward(input)
+val gradOut = Tensor(1, 2, 2, 2).fill(0.1f)
+val gradIn = m.backward(input,gradOut)
+
+scala> print(input)
+(1,1,.,.) =
+-0.6636712      -1.3765892      -1.51044
+0.4502934       -0.38438025     -0.4279503
+-1.5327895      -0.33594692     1.5972415
+
+[com.intel.analytics.bigdl.tensor.DenseTensor$mcF$sp of size 1x1x3x3]
+
+scala> print(output)
+(1,1,.,.) =
+-0.2903078      -0.5241474
+-0.17961408     -0.11239494
+
+(1,2,.,.) =
+-1.3147768      -2.3738143
+-0.81345534     -0.5090261
+
+[com.intel.analytics.bigdl.tensor.DenseTensor of size 1x2x2x2]
+
+scala> print(gradOut)
+(1,1,.,.) =
+0.1     0.1
+0.1     0.1
+
+(1,2,.,.) =
+0.1     0.1
+0.1     0.1
+
+[com.intel.analytics.bigdl.tensor.DenseTensor$mcF$sp of size 1x2x2x2]
```

Comment:
what is $mcF$sp?

Reply:
it means some specific version of class in scala. which is a part of class name when print the tensor